web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx

```diff
@@ -57,6 +64,11 @@ const HfiCalculatorPage: React.FunctionComponent = () => {
   const [previouslySelectedDateOfInterest, setPreviouslySelectedDateOfInterest] =
     useState(DateTime.now().setZone('UTC-7').toISO())
 
+  const weeklyViewAsString = RowManager.exportWeeklyRowsAsStrings(fireCentres, dailies)
```

Comment:
Should we only generate these strings when the copy button is clicked?

Reply:
Reworked - string is only generated when the copy button is clicked